backend/sozisel/lib/sozisel_web/schema/types/session_types.ex

```diff
@@ -65,6 +65,7 @@ defmodule SoziselWeb.Schema.Types.SessionTypes do
     field :event_id, non_null(:id)
     field :event_name, non_null(:string)
     field :submissions, non_null(:integer)
+    field :start_minute, non_null(:integer)
```

Comment:
I need `event_type` also. Sorry for changing my mind.

Reply:
Never mind, I can change it myself, focus on those three queries I suggest you add.